packages/sitecore-jss-angular/ngcc.config.js

```diff
@@ -1,7 +0,0 @@
-module.exports = {
```

Comment:
Seems like it's not needed. Locally it's working fine without this config. Need to check how it will work without symlink